### PR TITLE
[examples] create new example gcp-from-scratch

### PR DESCRIPTION
**Pull request by Z4ck404**
creating a gcp cluster from scratsh

### CODE REVIEW

File: examples/gcp-from-scratch/main.tf [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+################################################################################
+# Create networking
+################################################################################
+
+resource "google_compute_network" "this" {
+  name = "${var.cluster_name}-network"
+}
+
+
+resource "google_compute_subnetwork" "this" {
```
Comment (jrj-d) [inline examples/gcp-from-scratch/main.tf:16]: Does this mean this is a single-AZ cluster?

**Review by jrj-d — COMMENTED** [review]
This looks good to me, pending my question about subnets and AZs.

Just to make sure: have you been able to successfully run an app on the cluster you created? Thanks!

File: examples/gcp-from-scratch/main.tf (hunk 1 above)
Comment (Z4ck404) [reply examples/gcp-from-scratch/main.tf:16]: The cluster looks healthy in the spot UI but i didn't run an app on it yet. will try that and get back to you

Comment (jrj-d) [reply examples/gcp-from-scratch/main.tf:16]: > The cluster looks healthy in the spot UI but i didn't run an app on it yet. will try that and get back to you

We learn a lot but running an app on a cluster, especially about permissions and connectivity issues

Comment (Z4ck404) [reply examples/gcp-from-scratch/main.tf:16]: I could run the application but after creating and configuring a VNG manually !

**Review by jrj-d — APPROVED** [review]
Great job!

**Pull request merged**